apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.ts

```diff
@@ -11,7 +13,31 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     const institutions = await sqldb.queryRows(sql.select_institutions, InstitutionRowSchema);
-    res.send(AdministratorInstitutions({ institutions, resLocals: res.locals }));
+    const availableTimezones = await getAvailableTimezones();
+    res.send(
+      AdministratorInstitutions({ institutions, availableTimezones, resLocals: res.locals }),
+    );
+  }),
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    console.log(req.body);
```

Comment:
```suggestion
```

Reply:
Made this change locally in commit [fb604ae](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/fb604aec1a5e82dbf069cab10a5723c80a85a5c8).